library/Reporting/Web/Forms/SendForm.php

```diff
@@ -32,8 +35,35 @@ protected function assemble()
 
         (new SendMail())->initConfigForm($this, $this->report);
 
+        $radio = $this->addElement('radio', 'source_radio', [
+            'label'   => $this->translate('E-Mail Source'),
+            'options' => [
+                'manual'   => $this->translate('Manual input'),
+                'contacts' => $this->translate('Contacts'),
+            ],
+            'value'   => 'contacts',
```

Comment:
I think you should use `$this->getPopulatedValue('source_radio', 'contacts')` instead.